ProffieOS.ino

```diff
@@ -1496,6 +1495,142 @@ private:
 
 StaticWrapper<Parser<SerialAdapter>> parser;
 
+#ifdef TEENSYDUINO
+#include "common/_usb_rawhid.h"
+#if defined(RAWHID_INTERFACE)
+class RawHIDParser: Looper, StateMachine  {
```

Comment:
Why do we need to duplicate the whole parser?
Isn't it enough to use the existing parser with a different adapter?